server/README.md

```diff
@@ -137,7 +137,11 @@ You can install them by the following command:
     $ sudo apt-get install librabbitmq-dev rabbitmq-server
 
 ## How to build Hatohol
-First, you need to install required libraries.
+First, you need to install Django.
+
+    $ pip3 install django
```

Comment:
* Hatohol doesn't officially support Python 3 (probably no one uses Python 3)
* Should use same command in client/README.md to install django

BTW I want to enable to build it without Django in the future.